modules/apps/layout/layout-seo-service/src/main/java/com/liferay/layout/seo/internal/LayoutSEOLinkManagerImpl.java

```diff
@@ -75,6 +77,15 @@ public String getFullPageTitle(
 		String layoutTitle = _getPageTitle(
 			layout, portletId, tilesTitle, titleListMergeable,
 			subtitleListMergeable, locale);
+
+		LayoutSEOGeneralGroupConfiguration layoutSEOGeneralGroupConfiguration =
+			_configurationProvider.getGroupConfiguration(
+				LayoutSEOGeneralGroupConfiguration.class, layout.getGroupId());
+
+		if (!layoutSEOGeneralGroupConfiguration.showCompanyName()) {
```

Comment:
Shouldn't this be called `showSiteAndCompanyName` since it's affecting both?

Reply:
Alternatively, should we have two configurations... one for site and one for company?

Reply:
I've changed it to showOnlyLayoutTitle, is it ok for you?